### ISSUE TITLE
adm1 outdated states/provinces for Northern Ireland

### ISSUE THREAD

**Issue by MLWorley** [nvkelso/natural-earth-vector#766]
To quote statoids.com on this:
"2015-04-01: By the Local Government Act (Northern Ireland) 2014, Armagh, Banbridge and Craigavon district created from Armagh, Banbridge, and Craigavon districts; Antrim and Newtownabbey district created from Antrim and Newtownabbey districts; Causeway Coast and Glens district created from Ballymoney, Coleraine, Limavady, and Moyle districts; North Down and Ards district created from Ards and North Down districts; Derry and Strabane district created from Derry and Strabane districts; Fermanagh and Omagh district created from Fermanagh and Omagh districts; Lisburn and Castlereagh district created from Castlereagh and Lisburn districts; Mid and East Antrim district created from Ballymena, Carrickfergus, and Larne districts; Newry, Mourne and Down district created from Down and Newry and Mourne districts; Mid Ulster district created from Cookstown, Dungannon, and Magherafelt districts. Belfast district remained basically unchanged, except that, comparing maps of the divisions before and after this change, it can be seen that Belfast district acquired Belvoir, Cregagh, Gilanhirk, Hillfoot, and Merok wards from Castlereagh district and Dunmurry, Lagmore, Poleglass, and Twinbrook wards from Lisburn district. Also, Newry, Mourne and Down district acquired Ballyward and Castlewellan wards from Banbridge district. Armagh, Banbridge and Craigavon district acquired some territory from Dungannon district, not matching any ward."

The ISO 3166-2 subdivision codes for Northern Ireland were also redone in 2015 (https://www.iso.org/obp/ui/#iso:code:3166:GB, see the codes with the parent subdivision GB-NIR).

(Just looking at the ISO 3166-2 changes makes the mergers look simpler than they were, at first.)

The UK does appear to have an accessible reference dataset for their subdivisions as shapefiles at the following URL:

https://geoportal.statistics.gov.uk/datasets/ons::counties-and-unitary-authorities-december-2021-uk-bfc/explore?location=54.386028%2C-6.663118%2C9.06

Right now the ne_10m_admin_1_states_provinces divisions for Northern Ireland have ISO 3166-2 codes and names that are a little off; the last time I looked, the old Cookstown still had Cookstown's old ISO 3166-2 code but the name Mid Ulster, although none of the other changes to switch over to the Mid Ulster division looked as though they were present.

**Comment by marsmathis** [comment]
Agreed, this needs to be updated. But the last commit was three years ago, I’m not hopeful.

@nvkelso any chance of an update?
